cloud_tpu_colabs/JAX_demo.ipynb

```diff
@@ -252,7 +252,7 @@
     "def predict(params, inputs):\n",
     "  for W, b in params:\n",
     "    outputs = jnp.dot(inputs, W) + b\n",
-    "    inputs = jnp.tanh(outputs)\n",
+    "    outputs = jnp.tanh(outputs)\n",
```

Comment:
Same here

Reply:
This has been reverted in the latest commit.